tests/test_sota_checkpoints.py

```diff
@@ -466,46 +416,22 @@ def test_openvino_eval(self, openvino, sota_checkpoints_dir, sota_data_dir, ov_d
                          "{}/{config}.csv".format(config_folder, ov_data_dir, ir_model_folder,
                                                   PROJECT_ROOT, config=eval_test_struct.model_name_)
                 exit_code, err_str = self.run_cmd(ac_cmd, cwd=ACC_CHECK_DIR)
-
-        ac_metric = self.read_csv_metric(eval_test_struct.model_name_)
-        fp32_ac_metric = None
-        if eval_test_struct.reference_ is not None:
-            if os.path.exists('{}.csv'.format(eval_test_struct.reference_)):
-                fp32_ac_metric = self.read_csv_metric(eval_test_struct.reference_)
+                if exit_code != 0 or err_str is not None:
+                    assert False
+                assert True
             else:
-                fp32_ac_metric = self.ref_fp32_dict[str(eval_test_struct.reference_)]
-
-        if ac_metric:
-            diff_target = round((ac_metric - eval_test_struct.expected_), 2)
-            diff_fp32 = round((ac_metric - fp32_ac_metric), 2) if fp32_ac_metric is not None else None
+                assert False
         else:
-            diff_target = None
-            diff_fp32 = None
-
-        self.row_dict[eval_test_struct.model_name_] += self.make_ov_table_row(err_str,
-                                                                              ac_metric,
-                                                                              diff_target,
-                                                                              fp32_ac_metric,
-                                                                              diff_fp32)
-        retval = self.threshold_check(err_str,
-                                      diff_target,
-                                      eval_test_struct.diff_fp32_min_,
-                                      eval_test_struct.diff_fp32_max_,
-                                      fp32_ac_metric,
-                                      diff_fp32,
-                                      eval_test_struct.diff_target_min_,
-                                      eval_test_struct.diff_target_max_)
-
-        self.color_dict[eval_test_struct.model_name_], is_accuracy_within_thresholds = retval
-        assert is_accuracy_within_thresholds
```

Comment:
Where did all this code go? Where is the logic that checks the metrics against the thresholds now?

Reply:
The logic that checks thresholds for OpenVino is postponed in separate script in Jenkins trigger job.